libcodechecker/server/config_db_model.py

```diff
@@ -0,0 +1,93 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""
+SQLAlchemy ORM model for the product configuration database.
+"""
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from sqlalchemy import *
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import *
+
+CC_META = MetaData(naming_convention={
```

Comment:
Just as said below, the conventions could change. These are separate systems, that are possibly accessed and maintained through separate endpoints and code places.